doc/shelley-genesis.md

```diff
@@ -503,16 +652,315 @@ You'll notice that the automagic method has divided the total supply amongst
+issue counter must be kept with the operator's offline key. You'll notice they
+got created when we created the genesis delegate keys, or if you create a new
+stake pool key.
+
+Finally we have a confusing flag for the KES period. Each operational
+certificate specifies when the certificate is valid from. This is like a date
+but is specified in terms of KES periods, which is some number of slots long.
+Frankly, this needs improving in the CLI tools and/or documentation. For now
+we are creating a system from scratch so we can start with period 0.
+
+So lets go ahead and issue ourselves an operational certificate. We will sign
+using use the genesis delegate key we created earlier.
+```
+$ cardano-cli shelley node issue-op-cert \
+    --hot-kes-verification-key-file example/node1/kes.vkey \
+    --cold-signing-key-file example/delegate-keys/delegate1.skey \
+    --operational-certificate-issue-counter example/delegate-keys/delegate1.counter \
+    --kes-period 0 \
+    --out-file example/node1/cert
+```
 
 ## Starting a node
 
-TODO
+Now that we have generated our genesis.json, operational certificate, VRF signing key and KES signing key, we can run a node via:
+--TODO: What about creation of topology?
+--TODO: Probably need to add a default config.yaml for a shelley node
+--TODO: I needed to manuall update my time
+```
+$ cardano-node run \
+    --topology configuration/defaults/shelley/topology.json \
+    --shelley-kes-key example/KES/signkey \
+    --shelley-operational-certificate example/opcert \
+    --shelley-vrf-key example/VRF/signkey \
+    --config configuration/defaults/shelley/config.yaml \
+    --database-path example/db \
+    --port 3001 \
+    --socket-path example/socket/shelley-node
+```
+
+## Submitting a transaction
 
+Once you have a node up and running, you can submit transactions to it. You will need to:
+
+  1. Query the genesis UTxO
+  2. Create an address
+  3. Build a transaction
+  4. Sign the transaction
+  5. Submit the transaction
+
+#
+
+1. Querying the genesis UTxO. This will display the addresses and their funds that you defined in your genesis.json. It will allow you to get the transaction ids and their indexes to build the transaction.
+
+-- TODO: I had to create the socket file manually at this point and restart the node
+
+```
+cabal exec cardano-cli -- shelley query filtered-utxo \
+                        --address ${initialfundaddress} \
+                        --config configuration/defaults/shelley/config.yaml \
+                        --socket-path example/socket/shelley-node \
+                        --out-file /dev/null
+```
+You will get something like the following:
+
+```
+Filtered UTxO: UTxO (fromList [(TxIn (TxId {_TxId = 97681ef59a9d76c6db9247e7897d9e35c6f1c9d244ac233c0f30d02e2906549f}) 0,TxOut (Addr (KeyHashObj (KeyHash 97681ef59a9d76c6db9247e7897d9e35c6f1c9d244ac233c0f30d02e2906549f)) StakeRefNull) (Coin 1000000))])
```

Comment:
I think this output was changed in https://github.com/input-output-hk/cardano-node/pull/918